wiki/Article_Styling_Criteria/en.md

```diff
@@ -355,22 +436,135 @@ External links must be a clean and direct link to a reputable source with the li
+
+- e.g. `hardrock_mod_vs_easy_mod.jpg` (i.e. in English)
+- e.g. `hardrock_mod_vs_easy_mod-DE.jpg` (i.e. in German)
+- e.g. `hardrock_mod_vs_easy_mod-ZH-TW.jpg` (i.e. in Traditional Chinese)
+
+#### Screenshots of gameplay
+
+All screenshots of gameplay must be done in the stable build (not to be confused with the _stable (fallback)_ build), unless it is for a specific feature that is unavailable in the stable build (e.g. Target Practice).
+
+##### osu! client settings
+
+Note: If you do not want to change your current settings for the osu! wiki, you can install a separate installation of osu! in another directory!
+
+You must set these settings before taking a screenshot of the osu! client (settings not stated below are assumed to be at their defaults):
+
+-   Select language: `English`
+-   Prefer metadata in original language: `Enabled`
+-   Release stream: `Stable`
+-   Frame limiter: `VSync`
+-   Detect performance issues: `Disabled`
+-   Resolution: `1280x720`
+-   Fullscreen mode: `Disabled`
+-   Screenshot format: `JPEG (web-friendly)`
+-   Parallax: `Disabled`
+-   Menu tips: `Disabled`
+-   Seasonal backgrounds: `Never`
+-   Show thumbnails: `Enabled`
+-   Background dim: `30%`
+-   Progress display: `Top-Right (Pie)`
+-   Score meter type: `Hit error`
+-   Score meter size: `1x`
+-   Always show key overlay: `Enabled`
+-   Current skin: `Default` (first option)
+
+#### Image links
+
+Never use image links! Image links are links with the image as the link text. On a PC platform, they will work. However, on a mobile platform, they will not work as the pointer event will trigger the hover text, not the link.
 
 ---
 
-It is suggested to add a subfolder for the images (e.g. `img/` for all and any images). Images that are used in multiple different articles must go to the `/wiki/shared/` folder while those used for a single article are to be placed inside their folders for linking.
+Flag icons next to userlinks must be separate from the username. For example:
+
+```
+Do not: [![][flag_AU] peppy](/users/2)
+
+Do:    ![][flag_AU] [peppy](/users/2)
+```
+
+#### Flag icons
+
+*For a list of flag icons, see [issue \#328](https://github.com/ppy/osu-wiki/issues/328).*
+
+All of these icons uses the two letter code (all capital letters) and ends with the GIF extension. With a few exceptions being the multi-flag icons which are in the PNG format. When adding a flag inline, use this format:
+
+```
+![](/wiki/shared/flag/xx.gif)
+```
+
+`xx.gif` is the [ISO 3166-2](https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2) two-lettered country code for the flag.
+
+Adding the full country names in the alt text is optional.
 
 ### Tables
 
-Tables use the GFM syntax.
+Tables use the GFM syntax. Tables in the osu! wiki support headings along the first row.
```

Comment:
Can you note to use one space between `|` (i.e. `| Something |`) as one method. For beautifying, when a translator edits them, note that they have to fix the `beautifying`. That or use the one spacing method for consistency.

Hate the messed up spacing and the method itself of making it perfect in code. Prefer the one space method please.

Reply:
This might have to be discussed, unless people can do what they want..
I'd suggest

```markdown
| Header1 | Header2 | Header3 |
| ------- | ------- | ------- |
| content | longer content but still only one space | asdf |
```

and so on. Especially the row below the headers should be the same length as the header above in monospaced fonts. (one whitespace, X `-` characters, one whitespace; X being the amount of letters of the column header above)

Reply:
`| ------- | ------- | ------- |`
I suggest that this should be like:
`| :-- | --: | :-: | --- |` <- examples with align, should have three characters every time

Translations will have different width heading, so it should have a common format. Also, if all English files get fixed, the translators don't need to worry about anything.

Reply:
So the gist of this thread is that (with some of my thoughts):
- a vertical bar (`|`) must be on the left and right side
- never beautify tables
- the text of each cell must be padding with only one space on both sides
- the delimiter row must use three characters and can only looks like these:
  - `---` (for left align)
  - `:--` (for left align)
  - `--:` (for right align)
  - `:-:` (for centre align)
- the delimiter row must also be padded with only one space on both sides
- the left align must be done consistently

Side note: "delimiter row" is the row that separates the heading from the body and is also used for alignments.

So a table in raw text should look like this:
```
| Left Header1 | Another left header2 | Centre header3 | Right header4 |
| --- | :-- | :-: | --: |
| left | left, but with a colon | centred text is in here | right aligned text |
```

Correct?

Reply:
Yeah thats what I want it to be